Add S3 auth example to Kafka source (#892)
risingwavelabs/risingwave-docs: sql/commands/sql-create-source.mdx

- To monitoring backfill progress, use the [SHOW JOBS](/sql/commands/sql-show-jobs) command or check `Kafka Consumer Lag Size` in the Grafana dashboard (under `Streaming`).

> <Note>Even if the source is created with `scan.startup.mode` set to `latest`, the materialized view might still need to backfill data from the "latest" offset resolved when the source was initially created. If you want the materialized view to restart freshly from an existing source, you must drop and recreate the source to resolve a new start offset.</Note>

Reviewer: ```suggestion
<Note>Even if a source is created with `scan.startup.mode` set to `latest`, a materialized view may still backfill from the offset resolved at source creation time. To start from the current latest offset, drop and recreate the source.</Note>
```
